From 8c87b6aed048fbd6f8776fcfd58b24f5972e7b23 Mon Sep 17 00:00:00 2001
From: Raine Makelainen <raine.makelainen@jolla.com>
Date: Tue, 26 Mar 2019 15:43:30 +0200
Subject: [PATCH 16/17] Cleanup build configuration. Fixes JB#44612

[nemo][embedlite] Disable telemetry / health / sync services. Fixes JB#44612

Also Marionette is now disabled

Signed-off-by: Raine Makelainen <raine.makelainen@jolla.com>
---
 configure.in          |  2 +-
 xulrunner/confvars.sh | 12 ++++++------
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/configure.in b/configure.in
index a3ba8b2f6ae2..872b5b2cb347 100644
--- a/configure.in
+++ b/configure.in
@@ -8435,7 +8435,7 @@ fi
 dnl
 dnl Always build Marionette if not Android or B2G
 dnl
-if test "$OS_TARGET" != Android -a x"$MOZ_WIDGET_TOOLKIT" != x"gonk"; then
+if test "$OS_TARGET" != Android -a x"$MOZ_WIDGET_TOOLKIT" != x"gonk" -a x"$MOZ_WIDGET_TOOLKIT" != x"qt"; then
     AC_DEFINE(ENABLE_MARIONETTE)
 fi
 AC_SUBST(ENABLE_MARIONETTE)
diff --git a/xulrunner/confvars.sh b/xulrunner/confvars.sh
index 4b2131d083bf..eca5c51dcacd 100755
--- a/xulrunner/confvars.sh
+++ b/xulrunner/confvars.sh
@@ -5,18 +5,18 @@
 
 MOZ_APP_NAME=xulrunner
 MOZ_APP_DISPLAYNAME=XULRunner
-MOZ_UPDATER=1
+MOZ_UPDATER=
 MOZ_XULRUNNER=1
 MOZ_CHROME_FILE_FORMAT=omni
 MOZ_APP_VERSION=$MOZILLA_VERSION
 MOZ_EXTENSIONS_DEFAULT=" gio"
 MOZ_URL_CLASSIFIER=1
-MOZ_SERVICES_COMMON=1
-MOZ_SERVICES_CRYPTO=1
-MOZ_SERVICES_METRICS=1
-MOZ_SERVICES_SYNC=1
+MOZ_SERVICES_COMMON=
+MOZ_SERVICES_CRYPTO=
+MOZ_SERVICES_METRICS=
+MOZ_SERVICES_SYNC=
 MOZ_MEDIA_NAVIGATOR=1
 MOZ_USE_NATIVE_POPUP_WINDOWS=1
 MOZ_PLACES=
-MOZ_SERVICES_HEALTHREPORT=1
+MOZ_SERVICES_HEALTHREPORT=
 MOZ_DISABLE_EXPORT_JS=1
-- 
2.24.1

